From: Jason Michalski <armooo@armooo.net>
Date: Sat, 15 Mar 2014 23:55:14 -0400
Subject: Do not give an address to SysLogHandler

Let the default value of address to be used. This will work correctly on
more systems.

fixes #78
---
 cloudprint/cloudprint.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cloudprint/cloudprint.py b/cloudprint/cloudprint.py
index da840c1..14d9368 100755
--- a/cloudprint/cloudprint.py
+++ b/cloudprint/cloudprint.py
@@ -436,7 +436,7 @@ def main():
 
     # if daemon, log to syslog, otherwise log to stdout
     if args.daemon:
-        handler = logging.handlers.SysLogHandler(address='/dev/log')
+        handler = logging.handlers.SysLogHandler()
         handler.setFormatter(logging.Formatter(fmt='cloudprint.py: %(message)s'))
     else:
         handler = logging.StreamHandler(sys.stdout)
